editor/src/components/canvas/__snapshots__/ui-jsx-canvas-bugs.spec.tsx.snap

```diff
@@ -16,6 +16,16 @@ exports[`UiJsxCanvas #747 - DOM object constructor cannot be called as a functio
         position: absolute;
         background-color: rgba(255, 255, 255, 1);
         box-shadow: 0px 0px 1px 0px rgba(26, 26, 26, 0.3);
+        background-image: linear-gradient(
```

Comment:
<img width="909" alt="image" src="https://user-images.githubusercontent.com/2226774/180036273-a4bd0df1-fd97-4369-8488-725935ad13e9.png">
I think you murdered the github highlighter somehow

Reply:
This wasn't even a github thing - the regular `git diff` did the same! I have never seen this before.